src/newHome/homeOnboarding/hooks/useOnboardingData.tsx

```diff
@@ -176,14 +185,18 @@ const getStepsData = ({
 
 export const useOnboardingData = () => {
   const intl = useIntl();
+  const posthog = usePostHog();
   const { markOnboardingStepAsCompleted, onboardingState } = useOnboarding();
 
   const steps = getStepsData({
     intl,
     isStepCompleted: (step: OnboardingStepsIDs) => onboardingState.stepsCompleted.includes(step),
     onStepComplete: (step: OnboardingStepsIDs) => {
       markOnboardingStepAsCompleted(step);
+      posthog.capture("home_onboarding_step_complete_click", { step_id: step });
     },
+    posthogCapture: (step_id: string) =>
+      posthog.capture("home_onboarding_step_click", { step_id: step_id }),
```

Comment:
suggestion: I am not sure that we should use same event name, as someone can click mark as done an next cick primary action